### DISCUSSION TITLE
Possible to Load to a local WebServer?

### DISCUSSION THREAD

**Discussion by Lunchtime0614** [dvmarinoff/Auuki#127] [General]
Is it possible to load to a local webserver and then be accessed within the network via ip address of the webserver?  I'm wondering if just everything in the SRC folder is copied locally then would this work - are there dependencies to run locally?

**Comment by dvmarinoff** [comment]
Yes you can run it locally, there are no external dependencies, the project uses only what is already available in the browser. It's using plain simple vanilla javascript, web components, web bluetooth, web USB, web serial, web audio API etc.

Most of those APIs require https server to work. I am using nodejs light-server for development, but any other would work.

```
$ cd Flux/
$ npm install light-server
$ light-server -s ./src/ -p 8080
```

**Reply by Lunchtime0614** [reply]
Ahh I wasn’t running https originally and it wouldn’t allow the Bluetooth connections…made the switch and all is working now.
